test/test_nmt.py

```diff
@@ -1,21 +1,137 @@
 import time
 import unittest
 
+import can
 import canopen
+from canopen.nmt import NMT_STATES, NMT_COMMANDS
 from .util import SAMPLE_EDS
 
 
+class TestNmtBase(unittest.TestCase):
+    def setUp(self):
+        node_id = 2
+        self.node_id = node_id
+        self.nmt = canopen.nmt.NmtBase(node_id)
+
+    def test_send_command(self):
+        dataset = (
+            "OPERATIONAL",
+            "PRE-OPERATIONAL",
+            "SLEEP",
+            "STANDBY",
+            "STOPPED",
+        )
+        for cmd in dataset:
+            with self.subTest(cmd=cmd):
+                code = NMT_COMMANDS[cmd]
+                self.nmt.send_command(code)
+                self.assertNotEqual(self.nmt.state, "INITIALISING")
```

Comment:
Alternatively something more precise, kind of like this:

```suggestion
                self.assertEqual(self.nmt.state,
                                 NMT_STATES[COMMAND_TO_STATE[code]])
```

Reply:
Sure, seems simple enough and if it breaks, then something in the tested code must have gone wrong.

Reply:
> Sure, seems simple enough [...]

The PR as it stands, or the suggestion? :)

Reply:
This was related only to the suggestion.